<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionApp_Settings"/>
    <addaction name="actionDoc_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionNew_Slide_Group"/>
    <addaction name="actionEdit_Slide_Group"/>
    <addaction name="actionDelete_Slide_Group"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_DViz"/>
    <addaction name="separator"/>
    <addaction name="actionVisit_DViz_Website"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionImages_Import_Tool"/>
    <addaction name="actionText_Import_Tool"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Insert">
    <property name="title">
     <string>&amp;Insert</string>
    </property>
    <addaction name="actionImport_Slide_Group"/>
    <addaction name="actionAdd_PowerPoint_File"/>
    <addaction name="actionAdd_Group_Player"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menu_Insert"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dwItems">
   <property name="windowTitle">
    <string>Items</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="movable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tabSongs">
        <attribute name="title">
         <string>Songs</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabBible">
        <attribute name="title">
         <string>Bible</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabMedia">
        <attribute name="title">
         <string>Media</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabSlides">
        <attribute name="title">
         <string>Slides</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionNew_Slide_Group"/>
   <addaction name="actionEdit_Slide_Group"/>
   <addaction name="separator"/>
   <addaction name="actionSlide_Group_Properties"/>
   <addaction name="actionDelete_Slide_Group"/>
   <addaction name="separator"/>
   <addaction name="actionImages_Import_Tool"/>
   <addaction name="actionText_Import_Tool"/>
   <addaction name="actionAdd_PowerPoint_File"/>
   <addaction name="separator"/>
   <addaction name="actionToggle_Live_Output"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open ...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As ...</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionNew_Slide_Group">
   <property name="text">
    <string>New Slide Group</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDelete_Slide_Group">
   <property name="text">
    <string>Delete Slide Group</string>
   </property>
   <property name="shortcut">
    <string>Shift+D</string>
   </property>
  </action>
  <action name="actionAbout_DViz">
   <property name="text">
    <string>About DViz</string>
   </property>
  </action>
  <action name="actionVisit_DViz_Website">
   <property name="text">
    <string>Visit DViz Website</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionEdit_Slide_Group">
   <property name="text">
    <string>Edit Slide Group</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionSlide_Group_Properties">
   <property name="text">
    <string>Slide Group Properties</string>
   </property>
   <property name="shortcut">
    <string>Shift+F2</string>
   </property>
  </action>
  <action name="actionApp_Settings">
   <property name="text">
    <string>Program Settings ...</string>
   </property>
  </action>
  <action name="actionDoc_Settings">
   <property name="text">
    <string>Document Settings ...</string>
   </property>
  </action>
  <action name="actionImages_Import_Tool">
   <property name="text">
    <string>Images Import Tool</string>
   </property>
  </action>
  <action name="actionText_Import_Tool">
   <property name="text">
    <string>Text Import Tool</string>
   </property>
  </action>
  <action name="actionAdd_PowerPoint_File">
   <property name="text">
    <string>Add PowerPoint File ...</string>
   </property>
  </action>
  <action name="actionImport_Slide_Group">
   <property name="text">
    <string>Import Slide Group ...</string>
   </property>
  </action>
  <action name="actionAdd_Group_Player">
   <property name="text">
    <string>Add Group Player</string>
   </property>
  </action>
  <action name="actionToggle_Live_Output">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="dviz.qrc">
     <normaloff>:/data/stock-media-rec.png</normaloff>:/data/stock-media-rec.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Live Output</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="dviz.qrc"/>
 </resources>
 <connections/>
</ui>
